tree_test.go

```diff
@@ -1982,3 +1982,49 @@ func Benchmark_GetByIndex(b *testing.B) {
 		}
 	})
 }
+
+func TestNodeCacheStatisic(t *testing.T) {
+	db, err := db.NewDB("test", db.MemDBBackend, "")
+	require.NoError(t, err)
+	const numKeyVals = 100000
+	testcases := map[string]struct {
+		cacheSize int
+		expect    []int
```

Comment:
Instead of having this as `[]int`, can we split it into a field for each.
E.g.
```
expectFastHit int
expectFastMiss int
...
```

Currently, it's not intuitive which value is represented by which index in `expect`

Reply:
Done.